spec/models/vote_spec.rb

```diff
@@ -4,4 +4,21 @@
   context "associations" do
     it { should belong_to(:place) }
   end
+
+  context "validations" do
+    it "should make sure one user cannot vote more than once" do
+      uuid = SecureRandom.uuid
```

Comment:
I'm not sure I understand how this works: a user shouldn't be able to vote more than once on the same place, but shouldn't they be able to vote on as many places as they want?

Reply:
...but also, this was originally planned for Sprint 2, so we don't have to tackle it now.

Reply:
Oh, I'll update the spec title so it's clearer

Reply:
Gotcha, makes sense now. (Generating the `uuid` for the user will come later, right?)

Reply:
Yeah, that's right 👍